app/models/user.php

```diff
@@ -26,7 +25,25 @@ class User extends AppModel
         ),
         'confirmpassword' => array(
             'confirm'     => array('confirm_password'),
-        )
+        ),
+        'new_username'    => array(
+            'exist'       => array('validate_changes', 'username'),
+        ),
+        'new_email'       => array(
+            'exist'       => array('validate_changes', 'email'),
+        ),
+        'company'         => array(
+            'alphachars'  => array('validate_alpha'),
+            'length'      => array('validate_between', 1, 60),
+        ),
+        'division'        => array(
+            'alphachars'  => array('validate_alpha'),
+            'length'      => array('validate_between', 1, 30),
+        ),
+        'specialization'  => array(
+            'alphachars'  => array('validate_alpha'),
+            'length'      => array('validate_between', 1, 30),
+        ),
```

Comment:
Done po, I included po the table name as magic strings din po. :smile:  :bow:  https://github.com/renzosunico/MyClassroom/commit/af616b05b2ae0f497e6ee8913c847f3fb0ba753b https://github.com/renzosunico/MyClassroom/commit/1bdf42eed7d4a82b34f590aec10a9c7d32cb61e9